bindings/nodejs/code/stronghold_storage.ts

```diff
@@ -0,0 +1,93 @@
+import { NapiStronghold, NapiDID, NapiKeyLocation, NapiChainState, NapiIdentityState } from '../index.js'
```

Comment:
Nit: Can we rename the directory of this file (`code`) to something more specific, like `node`, `node-adapter` or `adapter` or something like that?